manifester/helpers.py

```diff
@@ -21,3 +21,14 @@ def simple_retry(cmd, cmd_args=None, cmd_kwargs=None, max_timeout=240, _cur_time
         time.sleep(_cur_timeout)
         response = simple_retry(cmd, cmd_args, cmd_kwargs, max_timeout, new_wait)
     return response
+
+def process_sat_version(sat_version, valid_sat_versions):
+    if len(sat_version) > 8:
```

Comment:
What is this length checking? What do 8 characters mean?

Reply:
This is a perhaps obscure way of checking whether the z-stream version is included in the `sat_version` config setting. The valid values for this parameter from the API's perspective are all 8 characters or less, e.g. `sat-6.11`. The value we're currently getting in Robottelo includes the z-stream version (e.g. `sat-6.12.0`). So I am assuming here that, if the `sat_version` config setting is longer than 8 characters, the z-stream version is being included.

I'm open to suggestions for other ways to check this. Alternatively, I could clarify the rationale for this check and the assumptions behind it in a comment.

Reply:
The check is efficient and should be pretty stable. A clarifying comment would help.

Another check could be to count the `.`'s in the version string